app/component/itinerary/navigator/NaviCardContainer.js

```diff
@@ -116,6 +128,10 @@ function NaviCardContainer(
     if (legChanged) {
       updateClient(topics, context);
       setCardExpanded(false);
+      setLegChanging(true);
+      setTimeout(() => {
+        setLegChanging(false);
+      }, 2000);
```

Comment:
Should we grab the timeout into a const and clear it in the useEffect's return function?